src/javascript/app/common/account_opening.js

```diff
@@ -77,9 +77,11 @@ const AccountOpening = (() => {
     const showResponseError = (response) => {
         getElementById('loading').setVisibility(0);
         getElementById('real_account_wrapper').setVisibility(1);
+        const error_message = response.error.message;
+        const client_error_message = response.error.code.message_to_client;
```

Comment:
it is better to check if `message_to_client` exists before accessing it.

Reply:
orite 👌